docs/api.rst

```diff
@@ -0,0 +1,37 @@
+===
+API
+===
+
+Similarly to Croud, we offer API to allow programmatic access to our Cloud
+products. API is operated using a key and a secret. These can be created in
+your account page:
+
+.. image:: _assets/img/create-api-key.png
+   :alt: Cloud Console New Api Key
+
+Click the *Generate new key* button to create your key. A popup with your
+key and secret will appear. Make sure to store your secret safely, as you
+cannot access it again. 
```

Comment:
I would also add that the key and secret can be used as HTTP Basic Auth credentials when calling the API. Maybe even worth giving an example:
```shell
$ curl -s -u $your_key:$your_secret https://console.cratedb.cloud/api/v2/users/me/
{"email":"some@example.com","hmac":"...","is_superuser":false,"name":"Some User","organization_id":"123","status":"active","uid":"uid","username":"some@example.com"}
```